l10n_fr_siret/__manifest__.py

```diff
@@ -12,7 +12,7 @@
     "website": "https://github.com/OCA/l10n-france",
     "license": "AGPL-3",
     "depends": ["l10n_fr", "base_view_inheritance_extension"],
-    "external_dependencies": {"python": ["python-stdnum>=1.18"]},
+    "external_dependencies": {"python": ["python-stdnum"]},
```

Comment:
You should keep it here also
```suggestion
    "external_dependencies": {"python": ["python-stdnum>=1.18"]},
```